docs/deprecations.md

```diff
@@ -384,9 +384,9 @@ Configure `httpApi.payload` explicitly to ensure seamless migration.
 
 Deprecation code: `OUTDATED_NODEJS`
 
-Support for Node.js v6 and v8 will be dropped with v2.0.0 release
+Support for Node.js v10 will be dropped with v3.0.0 release
```

Comment:
This message should remain valid for v1 users, which may also come to see documentation for this deprecation.

I'd probably reword it to:

```
Support for Node.js v8 was dropped with v2 release, while support for Node.js v10 will be dropped with v3 release
```